examples/TrussArc/dodo_trussarc.py

```diff
@@ -25,6 +25,7 @@
     # mesh parameters (meshing by layer height)
     "mesh_size_factor": 10,
     "number_of_layers": 10,
+    "is_close": False,
```

Comment:
Because if we set the flag to True, it means that the creator will connect the last point to the first point automatically (Close), which is not what we expected, as the point series provided stops at the last position and not going back to the origin. 

<img width="937" alt="image" src="https://github.com/BAMresearch/amworkflow/assets/102924717/389a6e63-be72-4f39-89a8-2ddae36a47ee">

Perhaps "is" is bad in this case. Maybe "Enclose" would be a better word?